packages/lesswrong/components/ea-forum/InstagramLandingPage.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { useMulti } from "../../lib/crud/withMulti";
+import sortBy from "lodash/sortBy";
+
+const styles = (theme: ThemeType): JssStyles => ({
```

Comment:
```suggestion
const styles = (theme: ThemeType) => ({
```
(needed to make `ClassesType<typeof styles>` work correctly)